src/volume.ts

```diff
@@ -656,4 +682,98 @@ export enum HostPathVolumeType {
    * A block device must exist at the given path.
    */
   BLOCK_DEVICE = 'BlockDevice'
-}
\ No newline at end of file
+}
+
+/**
+ * Options for an Nfs-based volume.
+ */
+ export interface NfsVolumeOptions {
+  /**
+   * path that is exported by the NFS server. More info: https://kubernetes.io/docs/concepts/storage/volumes#nfs
+   */
+  readonly path: string;
+
+  /**
+   * readOnly here will force the NFS export to be mounted with read-only permissions. Defaults to false. More info: https://kubernetes.io/docs/concepts/storage/volumes#nfs
+   *
+   * @default false. More info: https://kubernetes.io/docs/concepts/storage/volumes#nfs
```

Comment:
```suggestion
   * @see https://kubernetes.io/docs/concepts/storage/volumes#nfs
   * @default false
```

We use the `@default` annotation for automatic API reference generation, so nothing should be appended to it that doesn't relate to default behavior.